src/main/java/org/broadinstitute/dsm/model/ParticipantWrapper.java

```diff
@@ -246,6 +246,36 @@ else if (DBConstants.DDP_ABSTRACTION_ALIAS.equals(source)) {
         }
     }
 
+    public static Map<String, Map<String, Object>> getParticipantESDataConsideringNumberOfParameters(@NonNull DDPInstance instance, Map<String, String> filters, String source) {
+        Map<String, Map<String, Object>> participantESData;
+        String wholeFilter = filters.get(source);
+        String[] betweenAnds = wholeFilter.split(Filter.AND_TRIMMED);
+        boolean tooManyParameters = false;
+        participantESData = new HashMap<>();
+        String lessThanLimit = Arrays.stream(betweenAnds).filter(query -> query.split(Filter.OR_TRIMMED).length <= 900).collect(Collectors.joining(Filter.AND));
```

Comment:
ES query max is 1024 do we want to change the 900 to 1000? maybe make a parameter which can be re-used?

Reply:
okay, I'll change it to 1000